spec/unit/ogr/geometry/lwgeom_extensions_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 require 'ogr/geometry/lwgeom_extensions'
 
-RSpec.describe OGR::Geometry::LWGeomExtensions do
+RSpec.describe OGR::Geometry do
```

Comment:
Revert?

Reply:
I only changed because all of the other similar specs do this. If it was possible to test just the extensions by themselves, I'd lean more towards agreeing with you, but you have to create an `OGR::Geometry` to be able to do the tests, so I'd argue that we're really testing the `OGR::Geometry`, but I could see the argument the other way too. If you feel strongly about reverting, unless there's a good reason, I'd feel like we need to change the other specs too.

Reply:
Yeah, I don't disagree.